lib/mandrill_dm/message.rb

```diff
@@ -65,6 +65,10 @@ def merge_language
       return_string_value(:merge_language)
     end
 
+    def merge_vars
+      mail[:merge_vars] ? mail[:merge_vars].instance_variable_get('@value') : nil
```

Comment:
The `.value` does not contain the array as originally passed in, unfortunately -- see discussion here https://github.com/jlberglund/mandrill_dm/issues/17.